rhopart.F90

```diff
@@ -147,25 +150,24 @@ subroutine rhopart(carma, cstate, rc)
             h2o_vp=pvapl(iz, igash2o), temp=t(iz))
           if (rc < 0) return
 
-        else
-          ! rlow
+        else ! irhswell(igroup) /= I_WTPCT_H2SO4
```

Comment:
add another if for I_PETTERS, and then another else for fitzgerand and gerber

Reply:
Hi Mike, have you check if this has been added?

Reply:
Yes, this is added in @fvitt sandbox:

/glade/work/fvitt/camdev/carma_model_dev/src/physics/carma/base/rhopart.F90